### ISSUE TITLE
Cannot restart home assistant due to aarlo config

### ISSUE THREAD

**Issue by alsoszaa** [twrecked/hass-aarlo#795]
Logger: homeassistant.components.hassio
Source: components/hassio/__init__.py:568 
Integration: Home Assistant Supervisor (documentation, issues) 
First occurred: 3:16:54 PM (1 occurrences) 
Last logged: 3:16:54 PM

The system cannot restart because the configuration is not valid: Invalid config for [switch.aarlo]: [resources] is an invalid option for [switch.aarlo]. Check: switch.aarlo->resources. (See ?, line ?).

**Comment by JP13090** [comment]
Use the CTRL-F in your text editor as you go through all your yaml files.  Start with configuration, and maybe sensors.yaml too. 
 Find anything "aarlo" and verify it's removed properly if that's what started this, or installed correctly.  HA found it somewhere, so you can too.

**Comment by alsoszaa** [comment]
@JP13090 I have had this integration for a couple years and its been working fine. It was the latest update that broke the integration from allowing Home Assistant to restart. 
I have had this portion in the configuration.yaml for at least a year, but today I commented it out and it resolved the issue. If possible, you may need to look int this section causing config issues:
`#switch:
#  - platform: aarlo
#    siren: True
#    all_sirens: True
#    snapshot: True
#    doorbell_silence: True
#    siren_volume: 1
#    siren_duration: 10`

**Comment by twrecked** [comment]
Hi.

Sorry you're having issues. I used your configuration in my setup and it worked fine.  Would it be possible to paste more of the switch configuration here? The lines before and after?

**Comment by JP13090** [comment]
> @JP13090 I have had this integration for a couple years and its been working fine. It was the latest update that broke the integration from allowing Home Assistant to restart. I have had this portion in the configuration.yaml for at least a year, but today I commented it out and it resolved the issue. If possible, you may need to look int this section causing config issues: switch:
> 
> * platform: aarlo
>   siren: True
>   all_sirens: True
>   snapshot: True
>   doorbell_silence: True
>   siren_volume: 1
>   siren_duration: 10

Yep. Mine too. HA still restarted, but had lots of errors because of Aarlo.  I ended up removing the entire integration, including all my YAML.  Then, I started from scratch.  Works great now, including your platform/siren you posted.

Biggest factor here was 2FA authentication.  I now have that happening at the gmail account my Home Assistant logs into Arlo with.  Basically, I uninstalled everything, restarted HA, and then went through step by step, adding YAML a piece at a time.  I'm back in business here, 100% working.  Thanks twrecked!
